templates/repo/actions/view.tmpl

```diff
@@ -2,7 +2,15 @@
 
 <div class="page-content repository">
 	{{template "repo/header" .}}
-	<div id="repo-action-view" data-run-index="{{.RunIndex}}" data-job-index="{{.JobIndex}}" data-actions-url="{{.ActionsURL}}"></div>
+	<div id="repo-action-view"
+		data-run-index="{{.RunIndex}}"
+		data-job-index="{{.JobIndex}}"
+		data-actions-url="{{.ActionsURL}}"
+		data-locale-approve="{{.locale.Tr "repo.diff.review.approve"}}"
+		data-locale-cancel="{{.locale.Tr "cancel"}}"
+		data-locale-rerun="{{.locale.Tr "actions.rerun"}}"
+		>
```

Comment:
This distinct formatting screams `prettier` to me.
Did I guess correctly?

Reply:
I do use `prettier` but disabled it for my Gitea workspace... (maybe after years of using it I unconsciously format as it does) Running `make fmt` before committing is my workflow now.